src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -716,6 +717,24 @@ bool SQLPTRepresentation::SetMetaInfo(const std::string& ccpu_version) {
   return true;
 }
 
+bool SQLPTRepresentation::SetHardwareVersion(
```

Comment:
Where do you check return value of this function?(except tests)

Reply:
I don't check it, here I use common approach for such methods (but it may be wrong)

Reply:
If function returns bool value it should be checked somewhere.